app/models/video.rb

```diff
@@ -19,4 +19,20 @@ def self.search(search)
       scoped
     end
   end
+
+  #import videos to translate
+  #if return false, then youtube_id column/value missing in CSV file
+  def self.import(file)
+    CSV.foreach(file.path, headers: true) do |row|
+      if row['youtube_id'].nil?
+        return false
```

Comment:
You should be raising exceptions instead of simply returning false. In fact, you probably shouldn't return anything at all.